server/service/microsoft_mdm.go

```diff
@@ -579,3 +580,15 @@ func (svc *Service) GetAuthorizedSoapFault(ctx context.Context, eType string, or
 
 	return &soapFault
 }
+
+func (svc *Service) SignMDMMicrosoftClientCSR(ctx context.Context, subject string, csr *x509.CertificateRequest) ([]byte, string, error) {
+	// TODO: check if this method should require explicit authorization
```

Comment:
How do we want to handle authorization here? My understanding is that this method is not intended to be exposed as a standalone endpoint and would depend on auth checks performed by other endpoint handlers.

Reply:
yes, it will depend by auth checks performed by other endpoint handlers